pkg/network/node/node.go

```diff
@@ -311,38 +311,36 @@ func (node *OsdnNode) validateMTU() error {
 		return fmt.Errorf("unable to determine MTU while performing validation")
 	}
 
-	if int(node.networkInfo.MTU) > mtu+50 {
+	const MTUTaintKey string = "network.openshift.io/mtu-too-small"
+	n, err := node.kClient.CoreV1().Nodes().Get(node.hostName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("could not get Kubernetes Node object by hostname: %v", err)
+	}
+	mtuTooSmallTaint := &corev1.Taint{Key: MTUTaintKey, Value: "value", Effect: "NoSchedule"}
+	tainted := taints.TaintExists(n.Spec.Taints, mtuTooSmallTaint)
+	mtuWithOverhead := mtu + 50
+	if !tainted && int(node.networkInfo.MTU) > mtuWithOverhead {
 		resultErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
 			klog.V(2).Infof("Configured node MTU is more than default interface MTU plus VXLAN overhead, tainting node...")
```

Comment:
(This correctly describes what the old code was checking, but will need to be updated now.)